<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="requestTrButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TR요청</string>
    </property>
   </widget>
   <widget class="QListView" name="logListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="conditionComboBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>사용자 조건식</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accountComboBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌번호</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="monitoringButton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>조건검색</string>
    </property>
   </widget>
   <widget class="QListView" name="realDataListView">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="accountWindowButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌비밀번호</string>
    </property>
   </widget>
   <widget class="QComboBox" name="trComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continuousTrButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>연속조회</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1072</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
